src/charm.py

```diff
@@ -85,7 +94,7 @@ def _on_create_profile_action(self, event: ops.ActionEvent) -> None:
             combine_stderr=True,
         )
         try:
-            process.wait()
+            process.wait_output()
```

Comment:
I don't see that the code uses output, would there be anything in particular about this change? (just asking, i'm fine with this change :D)

Reply:
`wait_output` is needed to read stdout in `ExecError` or else the stdout is `None` in `ExecError`.